packages/datadog-instrumentations/src/handlebars.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+const shimmer = require('../../datadog-shimmer')
+const { channel, addHook } = require('./helpers/instrument')
+
+const handlebarsReadCh = channel('datadog:handlebars:compile:start')
+
+function wrapCompile (compile) {
+  return function () {
+    if (handlebarsReadCh.hasSubscribers) {
+      const source = arguments[0]
```

Comment:
i usually just use the named function argument, not sure why you're using the arguments var here ?